src/main/java/com/enterprise/eventmanagementsystem/dao/EventDAO.java

```diff
@@ -3,12 +3,9 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Qualifier;
 
-import java.util.HashMap;
-import java.util.Map;
 
 public class EventDAO implements IEventDAO {
 
-    Map<Integer, Event> allEvents = new HashMap<>();
 
```

Comment:
This is a just a bunch of unused imports and the map is being unused there is no need for these to continue to be implemented and is a poor coding practice.